templates/http/base/deployment.yaml

```diff
@@ -24,11 +24,19 @@ spec:
     spec:
       containers:
       - image:  {{values.appContainer}}
-        name: app-inference
+        name: app-interface
```

Comment:
Why `app-interface`?

Reply:
is it supposed to be called inference? I renamed it interface because the ENV's for this image are all called `APP_INTERFACE_CONTAINER`

Reply:
Huh, TIL. I always thought it was supposed to be `inference`, but didn't realize there were env vars with app interface.

What's app interface supposed to refer to here? CC @yangcao77

Reply:
Reverted the name.